### PR TITLE
Shadcn migration - PageHero

### PR DESCRIPTION
**Pull request by saurabhburade**
## Description

Migrate PageHero to tailwind.

### CODE REVIEW

File: src/components/PageHero.tsx [hunk 1]
```diff
@@ -48,113 +49,93 @@ const PageHero = ({
...
                       variant={button.variant}
                       href={button.href}
-                      onClick={() =>
-                        trackCustomEvent({
-                          eventCategory: button.matomo.eventCategory,
-                          eventAction: button.matomo.eventAction,
-                          eventName: button.matomo.eventName,
-                        })
-                      }
+                      customEventOptions={{
```
Comment (pettinarip) [inline src/components/PageHero.tsx:91]: Nice!

File: tailwind.config.ts [hunk 2]
```diff
@@ -102,7 +102,10 @@ const config = {
           800: "var(--orange-800)",
           900: "var(--orange-900)",
         },
-
+        text: {
```
Comment (pettinarip) [inline tailwind.config.ts]: Gonna remove these colors because we now default to white in the hero texts.

File: src/components/PageHero.tsx [hunk 3]
```diff
@@ -48,113 +49,93 @@ const PageHero = ({
...
+      )}
     >
-      <Box
-        maxW={{ base: "full", lg: "container.sm" }}
-        pt={{ base: isReverse ? 0 : 8, lg: 32 }}
-        pb={{ base: isReverse ? 8 : 0, lg: 32 }}
-        ps={{ base: 0, lg: 8 }}
-        me={{ base: 0, lg: 4 }}
+      <div
+        // TODO: container.sm width unavailable
```
Comment (pettinarip) [inline src/components/PageHero.tsx]: Removing comment since we will not have this measure (640px) on the new theme. We can treat it now as a custom size.

**Review by pettinarip — APPROVED** [review]
@saurabhburade LGTM! thanks

I've pushed a commit with minor style tweaks to align with latest design decisions.

File: src/components/PageHero.tsx (hunk 3 above)
Comment (saurabhburade) [reply src/components/PageHero.tsx]: Gotcha!! So every `container.sm` is considered as a custom size of `(640px)` .

Comment (pettinarip) [reply src/components/PageHero.tsx]: Yeah, we don't have that breakpoint on the theme anymore.

**Pull request merged**